packages/ds-healthcare-gov/src/components/Header/defaultMenuLinks.ts

```diff
@@ -44,16 +39,10 @@ export function defaultMenuLinks(options: DefaultMenuLinkOptions = {}) {
     hideLanguageSwitch,
     customLinksPassedIn,
   } = options;
-  const t = tWithLanguage(locale);
-  const isSpanish = languageMatches('es', locale);
+  const t = tWithLanguage(getLanguage());
```

Comment:
I think this could just be the imported `t` function ([like this module](https://github.com/CMSgov/design-system/blob/main/packages/design-system/src/components/Spinner/Spinner.tsx#L3)) now that we no longer pass any props as parameters

Reply:
This suggestion applies to a few other places in this change set too

Reply:
Oh wow yes this makes it a lot simpler. I didn't look closely enough at the `t` func.